tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -393,4 +393,61 @@ TEST_F(MatmulATenEvaluationTest, LinearWithBias) {
 
   EXPECT_TRUE(at::allclose(out[0], out_ref));
 }
+
+TEST_F(MatmulATenEvaluationTest, MatmulNodeConcrete) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+  std::vector<int64_t> a_shape{512, 32, 64}, b_shape{32, 1, 64, 128};
```

Comment:
Q: Do you have an example of parametrizing over sets of vectors using `TEST_P`. I was not able to make that work to test multiple input shape combinations.

Reply:
I do not but maybe the guru @wujingyue does.

Reply:
testing::Values?

Reply:
Yes, I was trying something like `testing::Values ((a_shape1, b_shape1) , (a_shape2, b_shape2))` but was running into errors. I'm looking into the documentation to see if there's any restriction on value types.

Reply:
I don't know what `(a_shape1, b_shape1)` means in C++. You might want to use `{}` or `std::make_pair` instead.

Reply:
I am using `std::make_tuple{{b, m, k}, {k, n}}` for input values.

Reply:
Is the problem that there are a variable number of dimensions in each test case?